mindsdb/api/executor/utilities/sql.py

```diff
@@ -147,6 +150,12 @@ def adapt_query(node, is_table, **kwargs):
             else:
                 if user_functions is not None:
                     user_functions.check_function(node)
+        # If a variable is used in the target list, add quotes around the value.
+        if isinstance(node, Variable):
+            # Variables in the targets will cause an error in DuckDB due to the '@@' or '@' prefix.
+            if kwargs.get('is_target'):
```

Comment:
is better to add is_target to arguments

Reply:
To arguments of `Variable`?

Reply:
I mean: add to arguments of function
`def adapt_query(node, is_table, is_target=None, **kwargs):`

Reply:
Got it. If we decide that this is required, I will add it in.